README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
```

Comment:
- [ ] Kindly customize the readme file, you can use this [template]( https://github.com/microverseinc/readme-template/blob/master/README.md) to create a nice looking readme file. This is just a template, please customize it to suit your need.